docs/docs/python/integrations/aws-lambda.mdx

```diff
@@ -66,6 +65,40 @@ Use [layout](../../../reference/targets/python_aws_lambda_function.mdx#layout) t
 `file` / `files` targets will not be included in the built AWS Lambda artifacts because filesystem APIs like `open()` would not load them as expected. Instead, use the `resource` and `resources` target. See [Assets and archives](../../using-pants/assets-and-archives.mdx) for further explanation.
 :::
 
+### Specifying a runtime explicitly
```

Comment:
Note that I'm still mulling on: this is currently within "Step 2", and maybe should be shifted out of the "happy flow" to the end. That is, leave it to the end as a reference for users who need more detail.

Reply:
If we treat this as a guide then I think putting it in step 2 logically makes sense. Users might as well understand the various ways in which they can select a runtime up front and choose what's appropriate for them.